<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Summary</class>
 <widget class="QDialog" name="Summary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>645</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>957</width>
    <height>645</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PoolMate Viewer</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QSplitter" name="main_splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>3</number>
     </property>
     <widget class="QTabWidget" name="tabs">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>280</height>
       </size>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="volume">
       <attribute name="title">
        <string>Volume</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="GraphWidget" name="volumeWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="technique">
       <attribute name="title">
        <string>Technique</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="GraphWidget" name="graphWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="lengths">
       <attribute name="title">
        <string>Lengths</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="GraphWidget" name="lengthWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QSplitter" name="lower_splitter">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>3</number>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Workouts</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QTableWidget" name="workoutGrid">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::ExtendedSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="rowCount">
               <number>1</number>
              </property>
              <property name="columnCount">
               <number>8</number>
              </property>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>60</number>
              </attribute>
              <attribute name="horizontalHeaderMinimumSectionSize">
               <number>30</number>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderDefaultSectionSize">
               <number>19</number>
              </attribute>
              <row/>
              <column>
               <property name="text">
                <string>Date</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Time</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Pool</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Duration</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Lengths</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Total m</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Cal</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Rest</string>
               </property>
              </column>
              <item row="0" column="0">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="0" column="1">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="0" column="2">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="0" column="3">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="0" column="4">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="0" column="5">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="0" column="6">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="0" column="7">
               <property name="text">
                <string/>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Sets</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QTableWidget" name="setGrid">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>60</number>
              </attribute>
              <attribute name="horizontalHeaderMinimumSectionSize">
               <number>30</number>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderDefaultSectionSize">
               <number>19</number>
              </attribute>
              <column>
               <property name="text">
                <string>Set</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Duration</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Lengths</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Dist</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Strokes</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Speed</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Effic</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Rate</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Stroke</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Rest</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Gap</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Lengths</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QTableWidget" name="lengthGrid">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>60</number>
              </attribute>
              <attribute name="horizontalHeaderMinimumSectionSize">
               <number>30</number>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderDefaultSectionSize">
               <number>19</number>
              </attribute>
              <column>
               <property name="text">
                <string>Length</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Time</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Strokes</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Style</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="buttonLayout2">
          <item>
           <widget class="QPushButton" name="editButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="printButton">
            <property name="text">
             <string>Print</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Graph:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="viewCombo">
            <item>
             <property name="text">
              <string>Sets</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>One week</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>One month</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Year by week</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Year by month</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget1">
       <layout class="QVBoxLayout" name="calendarLayout">
        <item>
         <layout class="QHBoxLayout" name="stroke_buttons">
          <item>
           <widget class="QCheckBox" name="efficCheck">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;SWOLF efficiency score&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Efficiency</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="strokeCheck">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Stroke length&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Stroke</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="rateCheck">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Stroke rate&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Rate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="speedCheck">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Speed = Seconds per 100m&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Speed</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="todayButton">
            <property name="text">
             <string>Today</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="CalendarWidget" name="calendarWidget">
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="firstDayOfWeek">
           <enum>Qt::Monday</enum>
          </property>
          <property name="gridVisible">
           <bool>true</bool>
          </property>
          <property name="horizontalHeaderFormat">
           <enum>QCalendarWidget::ShortDayNames</enum>
          </property>
          <property name="verticalHeaderFormat">
           <enum>QCalendarWidget::NoVerticalHeader</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="control_buttons">
          <item>
           <widget class="QPushButton" name="bestTimesButton">
            <property name="text">
             <string>Best Times</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="analysisButton">
            <property name="text">
             <string>Analysis</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="downloadButton">
            <property name="text">
             <string>Download...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="configButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Config...</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="closeButton">
            <property name="text">
             <string>Close</string>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphWidget</class>
   <extends>QWidget</extends>
   <header location="global">graphwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CalendarWidget</class>
   <extends>QCalendarWidget</extends>
   <header location="global">calendar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>todayButton</sender>
   <signal>clicked()</signal>
   <receiver>calendarWidget</receiver>
   <slot>showToday()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>948</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>631</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewCombo</sender>
   <signal>activated(int)</signal>
   <receiver>Summary</receiver>
   <slot>scaleChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>635</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>544</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calendarWidget</sender>
   <signal>clicked(QDate)</signal>
   <receiver>Summary</receiver>
   <slot>selectedDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>713</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>680</x>
     <y>544</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>downloadButton</sender>
   <signal>clicked()</signal>
   <receiver>Summary</receiver>
   <slot>syncButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>722</x>
     <y>636</y>
    </hint>
    <hint type="destinationlabel">
     <x>747</x>
     <y>544</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editButton</sender>
   <signal>clicked()</signal>
   <receiver>Summary</receiver>
   <slot>editButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>636</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>Summary</receiver>
   <slot>deleteClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>636</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>configButton</sender>
   <signal>clicked()</signal>
   <receiver>Summary</receiver>
   <slot>configButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>812</x>
     <y>636</y>
    </hint>
    <hint type="destinationlabel">
     <x>832</x>
     <y>544</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>Summary</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>948</x>
     <y>636</y>
    </hint>
    <hint type="destinationlabel">
     <x>942</x>
     <y>544</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>printButton</sender>
   <signal>clicked()</signal>
   <receiver>Summary</receiver>
   <slot>printButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>636</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>767</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>efficCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>Summary</receiver>
   <slot>onCheckClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>strokeCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>Summary</receiver>
   <slot>onCheckClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>641</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rateCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>Summary</receiver>
   <slot>onCheckClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>709</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>speedCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>Summary</receiver>
   <slot>onCheckClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>778</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>workoutGrid</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>Summary</receiver>
   <slot>workoutSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setGrid</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>Summary</receiver>
   <slot>setSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bestTimesButton</sender>
   <signal>clicked()</signal>
   <receiver>Summary</receiver>
   <slot>bestTimesButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>619</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>analysisButton</sender>
   <signal>clicked()</signal>
   <receiver>Summary</receiver>
   <slot>analysisButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>605</x>
     <y>621</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectedDate(QDate)</slot>
  <slot>selectedWorkout(int,int)</slot>
  <slot>startDate(QDate)</slot>
  <slot>endDate(QDate)</slot>
  <slot>syncButton()</slot>
  <slot>configButton()</slot>
  <slot>deleteClick()</slot>
  <slot>editButton()</slot>
  <slot>display()</slot>
  <slot>workoutSelected()</slot>
  <slot>scaleChanged(int)</slot>
  <slot>printButton()</slot>
  <slot>setSelected()</slot>
  <slot>onCheckClicked(bool)</slot>
  <slot>bestTimesButton()</slot>
  <slot>analysisButton()</slot>
 </slots>
</ui>
